omod/src/main/webapp/fragments/departmentFinanceSummaries.gsp

```diff
@@ -1,10 +1,22 @@
 <%
     ui.includeJavascript("financials", "jquery.dataTables.min.js")
     ui.includeCss("financials", "jquery.dataTables.min.css")
+    ui.includeJavascript("ehrconfigs", "knockout-3.4.0.js")
+    ui.includeJavascript("ehrcashier", "moment.js")
+    ui.includeJavascript("financials", "dataTables.buttons.min.js")
```

Comment:
in future we will have to move some of these files to ehrconfigs so that it can be used by other systems if needed